dockerfiles/keycloak-util/keycloak_config.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+echo "Configuring Keycloak..."
+
+cat /scripts/che-users-0.json.erb | \
+				    sed -e "/<% if scope.lookupvar('keycloak::che_keycloak_admin_require_update_password') == 'true' -%>/d" | \
```

Comment:
this breaks possibility to change configuration for admin user. 
this script should respect`CHE_KEYCLOAK_ADMIN_REQUIRE_UPDATE_PASSWORD` env var same as it is done in s2i
https://github.com/eclipse/che/blob/master/dockerfiles/init/s2i/keycloak/assemble#L52-L54

Reply:
Fixed